composer.json

```diff
@@ -33,5 +34,8 @@
         "branch-alias": {
             "dev-master": "1.3.x-dev"
         }
+    },
+    "suggest": {
+        "knplabs/rad-fixtures-load": "Allows to autoregister the dictionary provider as an alice provider"
```

Comment:
"Allows to autoregister the dictionary provider as an alice provider" This is the description of this componant ?